integration/test/app/basic/simple_test.go

```diff
@@ -92,7 +92,7 @@ func TestAppLifecycle(t *testing.T) {
 
 		var tarballPath string
 		{
-			tarballURL, err := appcatalog.GetLatestChart(ctx, key.DefaultCatalogStorageURL(), "chart-operator", "")
+			tarballURL, err := appcatalog.GetLatestChart(ctx, key.DefaultCatalogStorageURL(), "chart-operator", "0.13.0")
```

Comment:
Just to confirm. It's checking for the latest 0.13.X release? 

Maybe this needs a comment? As someone scanning the code quickly could get confused.

Reply:
Yes, it would check for the latest 0.13.x release in this case, so you are right, it could be confusing. I would add a comment, Thanks!